python/GafferUSD/PromoteInstances.gfr

```diff
@@ -0,0 +1,240 @@
+import Gaffer
```

Comment:
I'm not seeing any unit tests for this node, but there definitely seems to be enough subtlety in the implementation that we should have a good few.